### DISCUSSION TITLE
É possivel acessar pelo celular?

### DISCUSSION THREAD

**Discussion by MicaelSoares11** [MicaelSoares7/Profit-Armory-Pass-CS2-Calculator#3] [Ideas]
Como posso acessar o programa pelo celular?

**Comment by MicaelSoares7** [comment]
Sim, é possível!
Utilizando o app "Termux", ou qualquer editor de Python para dispositivos mobile.

**Comment by MicaelSoares7** [comment]
> Como posso acessar o programa pelo celular?
Sim, é possível!
Utilizando o app "Termux", ou qualquer editor de Python para dispositivos mobile.
